src/modules/enrollment/components/pages/ClientEnrollmentsPage.tsx

```diff
@@ -126,6 +126,18 @@ const ClientEnrollmentsPage = () => {
     type: 'EnrollmentsForClientFilterOptions',
   });
 
+  const rowLinkTo = useCallback(
+    (enrollment: ClientEnrollmentTableFields) => {
+      if (!enrollment.access.canViewEnrollmentDetails) return;
```

Comment:
Table looks a little funny when some enrollments are viewable (clickable) and others aren't. Could use some improvement to communicate why to the user. Maybe like we show the menu but the View Enrollment menu items is disabled with a lock icon or something?

Reply:
That seems reasonable to me! I don't think we have a pattern yet for the row primary action being disabled. But it does seem worth introducing something like that (rather than doing something hacky where non-viewable rows have a secondary action menu with the only item being a disabled version of the primary action)